### ISSUE TITLE
Opening from the browser fails

### ISSUE THREAD

**Issue by rnestler** [zrhoffman/f5vpn-arch#4]
When trying to connect to the vpn with the browser a pop-up shows up to open the link with f5-vpn, but nothing happens when I click "open link".

![f5-vpn](https://user-images.githubusercontent.com/1435346/87664498-a4e02a00-c765-11ea-9439-a5adee770c92.png)

Running from the command line as described in the README.rst worked though.

**Comment by rnestler** [comment]
When I manually select `/opt/f5/vpn/f5vpn` with Firefox it works. So it seems the .desktop file is somehow broken.

**Comment by zrhoffman** [comment]
Looks like `DBusActivatable=true` is breaking it. So, options would be adding a patch for `DBusActivatable=false` or adding a functioning systemd service file. The provided service file errors out with

```
Loaded: bad-setting (Reason: Unit com.f5.f5vpn.service has a bad unit file setting.)
```

**Comment by zrhoffman** [comment]
Fixed in ea6bd9040c. Thanks for reporting!
